tautulli/umbrel-app.yml

```diff
@@ -0,0 +1,64 @@
+manifestVersion: 1
+id: tautulli
+category: Media
+name: tautulli
+version: "247"
+tagline: Monitor your Plex Media Server
+description: >-
+  Tautulli is a 3rd party application that you can run alongside your Plex Media Server to monitor activity and track various statistics. Most importantly, these statistics include what has been watched, who watched it, when and where they watched it, and how it was watched. The only thing missing is "why they watched it", but who am I to question your 42 plays of Frozen. All statistics are presented in a nice and clean interface with many tables and graphs, which makes it easy to brag about your server to everyone else.
+  Features :
+  - Responsive web design viewable on desktop, tablet and mobile web browsers.
+  
+  
+  - Themed to complement Plex/Web.
+  
+  
+  - Easy configuration setup (no separate web server required).
+  
+  
+  - Monitor current Plex Media Server activity.
+  
+  
+  - Fully customizable notifications for stream activity and recently added media.
+  
+  - Top statistics on home page with configurable duration and measurement metric.
+  
+  
+  - Global watching history with search/filtering & dynamic column sorting.
+  
+  
+  - Full user list with general information and comparison stats.
+  
+  
+  - Individual user information including devices IP addresses.
+  
+  
+  - Complete library statistics and media file information.
+  
+  
+  - Rich analytics presented using Highcharts graphing.
+  
+  
+  - Beautiful content information pages.
+  
+  
+  - Full sync list data on all users syncing items from your library.
+  
+  
+  - And many more!!
+developer: JonnyWong16
+website: https://tautulli.com/
+dependencies:
+  - plex
```

Comment:
The container does not bind to any Plex data directories, nor does it require Plex to be running on the same server for the container to start, so we can remove Plex as a dependency.

It looks like when a user install the app they will be prompted to sign in to their Plex account and then point Tautulli to to the IP of their Plex server.